Tools/Scripts/webkitpy/style/checkers/jsonchecker.py

```diff
@@ -347,11 +347,15 @@ def check_property(self, property_name, value):
         keys_and_validators = {
             '*': self.validate_comment,
             'animatable': self.validate_boolean,
```

Comment:
It would be nice to rename this to 'animation-property' or something.

Reply:
Good idea. Done.